src/Internal/Symbols/pkg/Microsoft.Internal.Extensions.Symbols.csproj

```diff
@@ -0,0 +1,64 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <IsShipping>false</IsShipping>
+    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+    <PackageDescription>For internal-use only. This package contains symbols for Microsoft.Extensions projects.</PackageDescription>
+
+    <!--
+      This package is not meant to be used as a regular PackageReference. This is mean to transport
+      bits to the AspNetCore repo which builds a targeting pack with these assembies.
+    -->
+    <PackageType>Transport</PackageType>
+    <SymbolsPackagePath>data/symbols/$(TargetFramework)/</SymbolsPackagePath>
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+
+    <!-- The project representing the shared framework doesn't produce a .NET assembly or symbols. -->
+    <DebugType>none</DebugType>
```

Comment:
Amount of properties duplicated between project files make me sad. Move to refs.props?

Reply:
Targets too.